example/src/main/java/com/moengage/segment/example/AnalyticsKotlinActivity.kt

```diff
@@ -0,0 +1,82 @@
+package com.moengage.segment.example
+
+import android.app.TimePickerDialog
+import android.os.Bundle
+import android.widget.Button
+import android.widget.EditText
+import android.widget.TextView
+import android.widget.TimePicker
+import android.widget.Toast
+import androidx.appcompat.app.AppCompatActivity
+import com.segment.analytics.Analytics
+import kotlinx.serialization.json.buildJsonObject
+import kotlinx.serialization.json.put
+import java.text.DateFormat
+import java.text.SimpleDateFormat
+import java.util.*
+
+class AnalyticsKotlinActivity : AppCompatActivity() {
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setContentView(R.layout.activity_analytics_kotlin)
+        title = "Segment Analytics : Kotlin (Android)"
+
+        val analytics = (application as SampleApplication).analyticsKt
```

Comment:
This is not the right way to access a field declared in different class.

Reply:
FIXED